x-pack/plugins/timelines/public/components/t_grid/toolbar/fields_browser/helpers.tsx

```diff
@@ -78,40 +78,63 @@ export const getFieldCount = (category: Partial<BrowserField> | undefined): numb
  * Filters the specified `BrowserFields` to return a new collection where every
  * category contains at least one field name that matches the specified substring.
  */
-export const filterBrowserFieldsByFieldName = ({
+export function filterBrowserFieldsByFieldName({
```

Comment:
Can we have a linter rule for these? They keep ping-ponging back and forth. The style guide says to [prefer arrow functions](https://docs.elastic.dev/kibana-dev-docs/contributing/styleguide#prefer-modern-javascripttypescript-syntax)?

Reply:
Is there a reason it matters? Would you prefer a linter rule, or would you prefer to change the style guide?